packages/compiler-cli/ngcc/test/writing/new_entry_point_file_writer_spec.ts

```diff
@@ -678,7 +682,8 @@ runInEachFileSystem(() => {
               },
             ],
             ['fesm5', 'module']);
-        const packageJsonFromFile1 = JSON.parse(fs.readFile(packageJsonPath));
+        const packageJsonFromFile1 = JSON.parse(fs.readFile(packageJsonPath)) as
+            {fesm5_ivy_ngcc?: string, module_ivy_ngcc?: string};
```

Comment:
OMG in `ngcc/src/packages/entry_points.ts` we should create the following type:

```ts
export type NewEntryPointPropertiesMap = {
  [Property in PackageJsonFormatProperties as `${Property}_ivy_ngcc`]?: string;
};
```

And then this can be:

```suggestion
        const packageJsonFromFile1 = JSON.parse(fs.readFile(packageJsonPath)) as
            NewEntryPointPropertiesMap;
```